Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -99,13 +99,7 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
 
     let goToManagePledge = ctaButtonTapped
       .filter { canShowManageViewPledgeScreen($0.0, state: $0.2) }
```

Comment:
I think this (and other) signals here would read better with something like:

```swift
.map { project, _, state in (project, state) }
.filter(canShowManageViewPledgeScreen)
```

Reply:
I refactored all this 😅